ArduPlane/mode_qloiter.cpp

```diff
@@ -92,7 +92,7 @@ void ModeQLoiter::run()
 
     // call attitude controller with conservative smoothing gain of 4.0f
     attitude_control->input_euler_angle_roll_pitch_euler_rate_yaw(plane.nav_roll_cd,
-                                                                  plane.nav_pitch_cd,
+                                                                  MAX(plane.nav_pitch_cd, quadplane.q_fwd_nav_pitch_lim_cd),
```

Comment:
We should apply this up on 84, the `set_VTOL_roll_pitch_limit` call will be limiting to the wrong value other wise. So 84 would become:

`plane.nav_pitch_cd = MAX(loiter_nav->get_pitch(), quadplane.q_fwd_nav_pitch_lim_cd);`